### PR TITLE
ucs build for version greater 1.8

### PR DESCRIPTION
**Pull request by laclaro**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,13 +10,24 @@ endif
 #VERSION=1.6
 SRCDIRS=debian conffiles simplesamlphp-module-privacyidea/
 SRCFILES=Makefile
+DEFAULT_CONFFILES=conffiles-v1.9
+VERSION_NUMBER=$(shell echo ${VERSION} | sed 's/\.//;s@^[^0-9]*\([0-9]\+\).*@\1@')
+
+select-conffiles:
+	if [ ${VERSION_NUMBER} -ge 19 ]; then \
```
Comment (laclaro) [inline Makefile]: does not work for 1.10

**Review by spaceone — CHANGES_REQUESTED** [review]
Please don't use the variable `baseConfig` anymore. It's deprecated since ages, will be removed in UCS 5.0, it is necessary for the upgrade to UCS 5.0 that it is not already used anymore. The variable `configRegistry` can be used instead.

**Review by spaceone — COMMENTED** [review]
btw. UCR has boolean variables.
Things like `configRegistry.is_true('privacyidea/saml/verifyhost')` would be possible instead of checking strings manually. UCS Administrators are also used to set values like yes/on/1/etc. which are also covered by `is_true() / is_false()`

**Review by pablo-knight — APPROVED** [review]
next step build debfile

**Comment by spaceone** [comment]
@laclaro @pablo-knight Please don't ignore my review uncommented.

**Comment by laclaro** [comment]
@spaceone Thank's for your hint. I change the method in #9

**Pull request merged**